packs/feats/favor-of-heaven.json

```
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p>You're one of Hei Feng's chosen people and have the authority of the Celestial Court on your side. Once per day, you can cast a 6th-rank @UUID[Compendium.pf2e.spells-srd.Item.Spirit Blast] as a divine innate spell. In addition, you gain resistance 5 to both good and evil damage.</p>"
+            "value": "<p>You're one of Hei Feng's chosen people and have the authority of the Celestial Court on your side. Once per day, you can cast a 6th-rank @UUID[Compendium.pf2e.spells-srd.Item.Spirit Blast] as a divine innate spell. In addition, you gain resistance 5 to both holy and unholy damage.</p>"
```

```suggestion
            "value": "<p>You're one of Hei Feng's chosen people and have the authority of the Celestial Court on your side. Once per day, you can cast a 6th-rank @UUID[Compendium.pf2e.spells-srd.Item.Spirit Blast] as a divine innate spell. In addition, you gain resistance 5 to both holy and unholy.</p>"
```
Since holy/unholy aren't damage types we don't want to imply they are if we can without it sounding weird.